### ISSUE TITLE
[Accessibility]A11y_Java Tooling for IntelliJ/Eclipse_Eclipse Marketplace_Screen Reader: NVDA is not reading the link "2 matches. Browser for more solutions." 

### ISSUE THREAD

**Issue by soumyatippapuram** [microsoft/azure-tools-for-java#3368]
[Check out Accessibility Insights! ](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier!

GitHubTags: #A11yTCS; #A11yMAS; #DesktopApp; #Win10; #A11y_Java Tooling for IntelliJ/Eclipse-Stay Healthy; #MAS1.3.1; #Screen Reader; #C+E-H; #Java tooling for IntelliJ/Eclipse; #NVDA 

**Environment Details:**
Eclipse IDE for Enterprise Java Developers
Version: 2019-06 (4.12.0)
Build id: 20190614-1200

**OS Details:** 
OS: Windows 10 (Version 1903, Build: 18362.175)
MAS Violated: MAS 1.3.1 

**Repro Steps: ​**
1) Start NVDA and open Eclipse application
2) Tab till Help and select "Eclipse Marketplace" from the list
3) Search/Find for 'Azure Toolkit for Eclipse' and press enter
4) Tab till the link "2 matches. Browser for more solutions." and check if NVDA is reading it

**Actual Result: ​**
NVDA is not reading the link "2 matches. Browser for more solutions." 

**NOTE: Narrator is reading the link properly**

**Expected Result: ​**
NVDA should read the link "2 matches. Browser for more solutions." 

**User Impact: ​**
Screen reader users will not know where the focus is if NVDA doesn't read anything
 
**MAS Reference:**
[MAS 1.3.1](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4})

[Bug_3368_Eclipse_Marketplace_NVDA_Links.pptx](https://github.com/microsoft/azure-tools-for-java/files/3451863/Bug_3368_Eclipse_Marketplace_NVDA_Links.pptx)

**Comment by soumyatippapuram** [comment]
Reported a new issue https://bugs.eclipse.org/bugs/show_bug.cgi?id=551104 to Eclipse

**Comment by Flanker32** [comment]
Discussed this bug with Ryan and Sarah and approved to close this issue as its external and there is non blocking scenario.

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749769
